viz.yaml

```diff
@@ -227,6 +227,7 @@ publish:
       projection: "custom_projection"
       styles: "styles"
       map_utils: "map_utils"
+      rank_states_utils: "rank_states_utils"
```

Comment:
remove rank_states_utils from here (can just be listed as a depends of the key_stories viz item)